tests/smoke/events.js

```diff
@@ -44,7 +44,7 @@
 				editor = bot.editor,
 				resumeAfter = bender.tools.resumeAfter,
 				observer = observeSpellCheckEvents(editor),
-				starterHtml = '<p>asdf jkl dzxda</p>';
+				starterHtml = '<p>asdf jkl dzxda ^</p>';
```

Comment:
If no caret is provided, setHtmlWithSelection puts the caret in different places depending on browser, and can cause the test to fail in certain browsers.

There was a funny side effect where because of the wasteful bookmarking, this test somehow passed (because the selection would be somewhere near this arbitrarily created bookmark, and certain other things would be accidentally skipped)